.env.example

```
@@ -0,0 +1,9 @@
+# PostgreSQL Root Password (used by Docker image)
+POSTGRES_DB=intellibills
+POSTGRES_PASSWORD=super_secret_root_password
+
+# Intellibills Application Roles
+APP_RUNTIME_PASSWORD=runtime_password
+APP_WORKER_PASSWORD=worker_password
+APP_ADMIN_PASSWORD=admin_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_ADMIN_PASSWORD key should go before the APP_RUNTIME_PASSWORD key

(UnorderedKey)